rangecoder.h

```diff
@@ -217,11 +217,9 @@ namespace rangecoder
     class RangeDecoder : RangeCoder
     {
     public:
-        RangeDecoder() = delete;
-
-        RangeDecoder(std::queue<byte_t> bytes)
+        void start(std::queue<byte_t> bytes)
         {
-            m_bytes = bytes;
+            m_bytes = std::move(bytes);
 
```

Comment:
initialize range here
```suggestion
        m_lower_bound = 0;
        m_range = std::numeric_limits<range_t>::max();
```

Reply:
mark unresolved again, I think this is not resolved. Please give me reason you don't initialize range here.